protex/system.py

```diff
@@ -1,14 +1,15 @@
 import itertools
-import numpy as np
 import logging
 from collections import ChainMap, defaultdict, deque
-from simtk.openmm import openmm
+from typing import Dict
+
+import numpy as np
 
 logger = logging.getLogger(__name__)
 
 
 class IonicLiquidTemplates:
-    def __init__(self, states: list, allowed_updates: dict[frozenset]) -> None:
+    def __init__(self, states: list, allowed_updates: Dict[frozenset]) -> None:
```

Comment:
I think then we would need something like Dict[frozenset, Dict[str, float]] ?

Reply:
Or we leave it and wait until we know the final supported python versions and check later again?

Reply:
just realized that this doesn't work.
But is it not:
`Dict[str, frozenset]`?

Reply:
so, `str` for key and `frozenset` for value type?

Reply:
It looks like this:
allowed_updates = {}
allowed_updates[frozenset(["IM1H", "OAC"])] = {"r_max": 0.16, "delta_e": 2.33}

Reply:
ah, I see! yes, than your approach is the right one. can you add it?